### ISSUE TITLE
Automatic Keyboard Shortcuts

### ISSUE THREAD

**Issue by mikjsmith** [salvadorbs/Asuite#152]
**Describe the solution you'd like**
Functionality as per standard Windows menu systems where you press Alt-[menu letter] to access a menu, then a single keyboard shortcut (e.g. K) to access the function. This is as opposed to a complex multi-key stroke assignment

In this case ASuite automatically assigns a single character to access the programs from the Asuite menu. This is something PStart can do; one feature that would be useful is t he ability to override the selected character and assign your own.

**Comment by salvadorbs** [comment]
Do you mean something like that? 

![image](https://user-images.githubusercontent.com/3731721/143068168-0dab9963-90ce-472c-86a3-f69fd86b6289.png)

Being something managed by Windows itself, I don't think it's possible to change its behavior.

**Comment by mikjsmith** [comment]
Yes, this is what I was expecting. So it might be possible to add it as a standard feature, but not change the key?

This is the example from PStart

![image](https://user-images.githubusercontent.com/3380838/143070287-19a95318-e513-4ca4-9ad0-ca72b23bb224.png)
